pinc/LPage.inc

```diff
@@ -315,20 +315,23 @@ class LPage
     function markAsBad( $user, $reason )
     // Return TRUE if this report causes project to be marked bad.
     {
-        global $code_url, $site_abbreviation, $PAGE_BADNESS_REASONS;
+        global $code_url, $PAGE_BADNESS_REASONS;
 
         Page_markAsBad( $this->projectid, $this->imagefile, $this->round, $user, $reason );
 
         // Advise PM that the page has been marked bad.
-		$body_blurb =
-"Page {$this->imagefile} of this project has been marked bad due to {$PAGE_BADNESS_REASONS[$reason]}.
-Please visit
-    $code_url/tools/project_manager/handle_bad_page.php?projectid={$this->projectid}&image={$this->imagefile}
-to make any needed changes and make the page available for proofreading again.
-Until this report has been resolved, the project will not be able to leave
-the current round. If 10 pages are marked bad by at least 3 different users,
-the project will automatically be made unavailable.";
-        maybe_mail_project_manager($this->projectid, $body_blurb, "$site_abbreviation: Bad Page");
+        $project = new Project($this->projectid);
+        configure_gettext_for_user($project->username);
+        $body_blurb_messages[] = sprintf(_('Page %1$s of this project has been marked bad due to %2$s.'), $this->imagefile, $PAGE_BADNESS_REASONS[$reason]);
+        // TRANSLATORS: %s is a url
+        $body_blurb_messages[] = sprintf(
+            _("Please visit %s to make any needed changes and make the page available for proofreading again."),
+            "$code_url/tools/project_manager/handle_bad_page.php?projectid={$this->projectid}&image={$this->imagefile}");
+        $body_blurb_messages[] = _("Until this report has been resolved, the project will not be able to leave the current round.");
+        $body_blurb_messages[] = _("If 10 pages are marked bad by at least 3 different users, the project will automatically be made unavailable.");
+        $body_blurb = implode("\n", $body_blurb_messages);
```

Comment:
After looking at a sample email, I think we should use two "\n"s for all of these implodes to have an empty line between each one.

Reply:
For example, here is the format of the current email as output on TEST with the recipient's language being French (in a real email, the client will wrap the lines):
```
To: 
Subject: DPT: Bad Page: "Test and Consequences (Lincoln)"
Reply-To: dphelp@pgdp.net
From: dphelp@pgdp.net
Bonjour cpeel-peon9,
Ceci est un message automatique du site DP Test Site.

Regarding project "Test and Consequences (Lincoln)":
(projectID45b672b501229)
https://www.pgdp.org/~cpeel/c/project.php?id=projectID45b672b501229

Page 119.png of this project has been marked bad due to Missing Image.
Please visit https://www.pgdp.org/~cpeel/c/tools/project_manager/handle_bad_page.php?projectid=projectID45b672b501229&image=119.png to make any needed changes and make the page available for proofreading again.
Until this report has been resolved, the project will not be able to leave the current round.
If 10 pages are marked bad by at least 3 different users, the project will automatically be made unavailable.

Thank you!
DP Test Site
```

Reply:
I completely agree.